CHANGELOG.md

```diff
@@ -1100,15 +1291,22 @@
 * Ruin09
 * Son Bui
 * Suraj Banakar(बानकर) | スラジ
+* Yuan (Terry) Tang
```

Comment:
Hmm I wonder why this shows up in addition to "Yuan Tang" below

Reply:
It's probably because you had a different name in your commit author previously. Since the script uses pure `git` (no GitHub to do, for instance, email -> GH user), it pulls entirely from the commit metadata in the repo

Reply:
Is there a way to hack/merge these different ones?

Reply:
Not without making the script even hackier than it already is 😬 
I would prefer to use GH IDs where possible, but that would require some non-trivial work to do

You'd effectively have to make a dictionary of author names to reference and choose one by some preference (I guess latest one), which means an associative array in Bash, which requires Bash 4+. So another loop and a dictionary lookup. 

I did notice this in the diff, but thought it was nbd, especially since this PR collapses the Contributors section anyway. Your name here though, so a bit different I get it

Reply:
> You'd effectively have to make a dictionary of author names to reference and choose one by some preference (I guess latest one), which means an associative array in Bash, which requires Bash 4+. So another loop and a dictionary lookup.

Oh wait, the author name and email are the (non-unique) identifier, so you'd actually need an associative array for old name to new name (which there could be multiple matches for and which would be difficult to tell). In other words, hacky 🙃

Reply:
IMO, I'd punt this until we can do GH ID lookups, but that's not perfect either with regard to historical emails/IDs. Which, to be explicit, probably wouldn't happen anytime soon given the effort to impact ratio